From 848ae37277f45dcb5d5a9a87b56067f4ce582855 Mon Sep 17 00:00:00 2001
From: Andrey Prokopenko <prokopenko.rower@mail.ru>
Date: Tue, 24 Dec 2019 11:59:29 +0300
Subject: [PATCH] removed flag for building by g++9

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 017fe8675..88948309d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -282,7 +282,7 @@ if(FAIL_ON_WARNINGS)
   if(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /WX")
   else() # assume GCC
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
   endif()
 endif()
 
-- 
2.17.1

